src/LuigisBoxBundle/Response/UpdateByQuery/UpdateByQueryStatusResponse.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\LuigisBoxBundle\Response\UpdateByQuery;
+
+use Answear\LuigisBoxBundle\Response\ApiResponseError;
+
+class UpdateByQueryStatusResponse
+{
+    private const TRACKER_ID = 'tracker_id';
+    private const STATUS = 'status';
+    private const UPDATES_COUNT = 'updates_count';
+    private const FAILURES_COUNT = 'failures_count';
+    private const FAILURES = 'failures';
+    private const STATUS_COMPLETED = 'complete';
+
+    /**
+     * @var array
+     */
+    private $rawResponse;
+
+    /**
+     * @var string
+     */
+    private $trackerId;
+
+    /**
+     * @var bool
+     */
+    private $completed;
+
+    /**
+     * @var int|null
+     */
+    private $okCount;
+
+    /**
+     * @var int|null
+     */
+    private $errorsCount;
+
+    /**
+     * @var ApiResponseError[]|null
+     */
+    private $errors;
+
+    public function __construct(array $response)
+    {
+        $this->rawResponse = $response;
+        $this->trackerId = $response[self::TRACKER_ID];
+        $this->completed = self::STATUS_COMPLETED === $response[self::STATUS];
+
+        if ($this->completed) {
+            $this->okCount = $response[self::UPDATES_COUNT];
+            $this->errorsCount = $response[self::FAILURES_COUNT];
+            $this->errors = [];
+            foreach ($response[self::FAILURES] as $urk => $failure) {
+                $this->errors[] = new ApiResponseError($urk, $failure);
+            }
+        }
+    }
+
+    public function getRawResponse(): array
+    {
+        return $this->rawResponse;
+    }
+
+    public function getTrackerId(): string
```

Comment:
Is `trackerId` the same thing as `jobId` from `UpdateByQueryResponse`?

Reply:
No, `jobId` point to the currently processing job. TrackerId is not necessary, but the API returns it here. This is the client's public key.